src/bindings/python/src/pyopenvino/core/common.hpp

```diff
@@ -69,6 +69,8 @@ void fill_tensor_from_strings(ov::Tensor& tensor, py::array& array);
 
 void fill_string_tensor_data(ov::Tensor& tensor, py::array& array);
 
+const char* find_first_not_null(const char* ptr, size_t itemsize);
```

Comment:
I think is not required now as it is used as common `local` helper the header can be updated when it will be required in other files.